
<html>
<body>
<h3>Code(/Users/psksvp/Workspace/temp/temp/sv-benchmarks-svcomp16/c/loop-acceleration/functions_true-unreach-call1.c,false,clang-3.7,20)</h3><hr>
<pre>extern void __VERIFIER_error() __attribute__ ((__noreturn__));

void __VERIFIER_assert(int cond) {
  if (!(cond)) {
    ERROR: __VERIFIER_error();
  }
  return;
}

unsigned int f(unsigned int z) {
  return z + 2;
}

int main(void) {
  unsigned int x = 0;

  while (x < 0x0fffffff) {
    x = f(x);
  }

  //__VERIFIER_assert(!(x % 2));
  if(x % 2) __VERIFIER_error();
}
</pre><hr>
<pre>UNKNOWN
refinement failure due to exception: main is not verifiable, calls to the following functions were not inlined: f</pre><hr>
</body>
</html>
      